frontend/src/Components/Filter/Builder/MinimumAvailabilityFilterBuilderRowValue.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import FilterBuilderRowValue from './FilterBuilderRowValue';
+
+const protocols = [
+  { id: 'announced', name: 'Announced' },
+  { id: 'inCinemas', name: 'In Cinemas' },
+  { id: 'released', name: 'Released' }
```

Comment:
Should these be translated?

`src\NzbDrone.Core\Localization\Core\en.json` is where you would check for existing keys or add new ones as needed

Reply:
I just sort of copied what was already specified in: frontend/src/Components/Filter/Builder/ReleaseStatusFilterBuilderRowValue.js

I will add the translation there as well.